cmd/buildah/list.go

```diff
@@ -10,7 +10,7 @@ import (
 var (
 	listFlags = []cli.Flag{
 		cli.BoolFlag{
-			Name:  "quiet",
+			Name:  "quiet, q",
```

Comment:
That sounds good to me.  Basically you want to add an option though correct, since there will still be a quiet option.  You just want to change the meaning.